requirements.txt

```diff
@@ -1,5 +1,5 @@
 torch>=1.9.*
 torchmetrics>=0.4.1
 pytorch-lightning>=1.6.0
-lightning-utilities==0.3.0
+lightning-utilities==0.4.1
```

Comment:
```suggestion
lightning-utilities==0.4.1, !=0.4.0
```

Reply:
Since we are pinning to specific version, the `!=0.4.0` is not really needed